CSS/styles.css

```diff
@@ -38,7 +36,11 @@ body {
   align-items: center;
 }
 
-.htitle {
+.headtitle {
+  width: 100%;
+}
```

Comment:
Please use flexbox here to layout the element.
This is part of the requirement
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/7243132/170834123-3fcd9fff-512f-4378-bb8c-ace7ea441760.png">